tests/layer_tests/onnx_tests/test_conv.py

```diff
@@ -481,7 +481,7 @@ def test_conv_5D_precommit(self, params, dilations, pads, strides, bias, ie_devi
             *self.create_net(**params, shape=[2, 3, 25, 25, 25], dilations=dilations, pads=pads,
                              strides=strides,
                              bias=bias, ir_version=ir_version),
-            ie_device, precision, ir_version, temp_dir=temp_dir, use_old_api=use_old_api)
+            ie_device, precision, ir_version, temp_dir=temp_dir, use_old_api=use_old_api, custom_eps=1e-1)
```

Comment:
why is the threshold is so big: 0.1 that is not normal for convs or any layers

Reply:
Hi Roman!
I changed pr a little bit
regarding your question - based on comments in the ticket from @sshlyapn for test_conv_5d_precommit test with the next data {"ie_device": "GPU",  "precision": "FP16"
We should increase the threshold
Copy of comment:
"""
The issue is not connected with GPU Plugin itself, there are just a very rare cases where difference between results and ref values exceed expected threshold due to FP16 data types calculation. So, test should be fixed
"""

Reply:
Hi @andriipx, I understand.
Do we have the same problem with other plugins like CPU, etc? Such threshold looks not valid. I see no difference between conv1d, conv2d, etc.

@sshlyapn, please comment on it.

Thanks,
Roman

Reply:
I see your point. On CPU we have no such problem because anyway it computes in FP32 format, right?  Why would we like to increase the current error threshold, if the reproduction rate is too low?
Now we can increase this threshold only for GPU without affection CPU.

Reply:
Yeah, as for CPU it looks like so. Although reproduction is very rare CI team face this divergence sometime... And that is why I suggested to change threshold for this particular test and only for GPU as you mentioned

Reply:
Yes, please set up this threshold only for GPU device